udata/core/dataservices/models.py

```diff
@@ -31,6 +31,19 @@ def hidden(self):
                     db.Q(deleted_at__ne=None) |
                     db.Q(archived_at__ne=None))
 
+class HarvestMetadata(db.DynamicEmbeddedDocument):
```

Comment:
I think I create a new one because the old one seems a little bit legacy… Maybe we could talk about each field and see if they're still required (it seems to me there is a lot of duplicate or useless information…)

For exemple: two `archived`? `remote_id` and `dct_identifier` are the same thing, no? What is `uri`? `created_at` for the harvest metadata is which creation date? etc. :-)

```
    backend = db.StringField()
    created_at = db.DateTimeField()
    modified_at = db.DateTimeField()
    source_id = db.StringField()
    remote_id = db.StringField()
    domain = db.StringField()
    last_update = db.DateTimeField()
    remote_url = db.URLField()
    uri = db.StringField()
    dct_identifier = db.StringField()
    archived_at = db.DateTimeField()
    archived = db.StringField()
```

Reply:
I think we should try to improve if we can because we have 5 different `last_update` and nobody knows which one is which… `last_harvested_at` is much clearer… But if you think it's really important I can revert this change

Reply:
Done in 30ba48d17